zetaclient/hsm_signer_test.go

```diff
@@ -0,0 +1,72 @@
+package zetaclient
+
+import (
+	"crypto/rand"
+	"log"
+	"testing"
+
+	"github.com/frumioj/crypto11"
+
+	btcsecp256k1 "github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	"github.com/stretchr/testify/require"
+	keystone "github.com/zeta-chain/keystone/keys"
+)
+
+func TestSignSecp256k1(t *testing.T) {
+	//Skipping test as it isn't guaranteed that an HSM will be configured
+	t.SkipNow()
```

Comment:
I don't get it

Can't we configure it for tests?

Reply:
I meant that it isn't guaranteed that the host machine will have an HSM connected and configured.

Reply:
There isn't some way to mock it for testing purposes?

Reply:
Ok
Instead using `SkipNow`, could we using a custom build tag instead? Like `+build hsm-test`
And have a command `make test-hsm` to execute this test?